src/Codeception/Lib/ParamsLoader.php

```diff
@@ -78,53 +80,29 @@ protected function loadYamlFile()
         return $params;
     }
 
-    protected function loadXmlFile()
+    protected function loadIniFile()
     {
-        $paramsToArray = function (\SimpleXMLElement $params) use (&$paramsToArray) {
-            $a = [];
-            foreach ($params as $param) {
-                $key = isset($param['key']) ? (string) $param['key'] : $param->getName();
-                $type = isset($param['type']) ? (string) $param['type'] : 'string';
-                $value = (string) $param;
-                switch ($type) {
-                    case 'bool':
-                    case 'boolean':
-                    case 'int':
-                    case 'integer':
-                    case 'float':
-                    case 'double':
-                        $a[$key] = settype($value, $type);
-                        break;
-                    case 'constant':
-                        $a[$key] = constant($value);
-                        break;
-                    case 'collection':
-                        $a[$key] = $paramsToArray($param);
-                        break;
-                    default:
-                        $a[$key] = (string) $param;
-                }
-            }
-
-            return $a;
-        };
+        return parse_ini_file($this->paramsFile);
+    }
 
-        return $paramsToArray(simplexml_load_file($this->paramsFile));
+    protected function loadPhpFile()
```

Comment:
Was it really necessary to reorder methods?
It would be much easier to merge this PR if the only affected method was `loadDotEnvFile`.

Reply:
Sorry for that, that was my auto formatter I will update the PR

Reply:
@Naktibalda Please let me know if you want me to change more!